concrete_datastore/api/v1/validators.py

```diff
@@ -1,7 +1,58 @@
 # coding: utf-8
+from concrete_datastore.concrete.constants import EMPTY_VALUES_MAP
 from django.core.exceptions import ValidationError
 
 
 def validate_file(fieldfile_obj):
     if fieldfile_obj.file is None:  # skip-test-coverage
-        raise ValidationError("Erreur : fichier absent.")
+        raise ValidationError("Error : missing file.")
+
+
+def validate_required(field):
```

Comment:
Naming must be explicit. In this case you return a function. So name it like `get_validate_field_function`

Reply:
ou plutôt `get_field_validator`